man/fi_tcp.7.md

```diff
@@ -32,6 +32,16 @@ The following features are supported
 *Shared Rx Context*
 : The tcp provider supports shared receive context
 
+# PROVIDER SPECIFIC OPERATION FLAGS
+: The tcp provider supports the following op flags
+
+*FI_FLAG_TCP_NO_CONNECT*
+: This flag indicates that operations should fail if there is not an
+  existing connection to the remote peer.  Instead, if there is not
+  an established connection, an FI_ENOTCONN error should be expected.
```

Comment:
`Instead ..... should be expected.` --> `In such case, an FI_ENOTCONN error should be returned.`